gcc/configure.ac

```diff
@@ -4778,7 +4837,7 @@ foo:
     ;;
 
 changequote(,)dnl
-  i[34567]86-*-* | x86_64-*-* | aarch64-*-*)
+  i[34567]86-*-* | x86_64-*-*)
```

Comment:
This line was doing nothing becase the topmost case contains `aarch64-*-*` handler already. The point of the fix is to copy&paste the relevant checks from this handler to the upper one.